app/assets/stylesheets/users.scss

```diff
@@ -46,3 +49,15 @@ $red: #ff0000;
     margin: 0 0 30px;
   }
 }
+
+.col.l3.m5.s5.user-show {
+  height: 30em;
+  background-color: #a3cded;
+}
+
+.col.l9.m7.s7.user-show {
+  height: 30em;
+  background-color: #edc3a3;
```

Comment:
Color literals like `#edc3a3` should only be used in variable declarations; they should be referred to via variable everywhere else.